src/SpotifyApi.NetCore/FollowApi.cs

```diff
@@ -134,14 +134,193 @@ public FollowApi(HttpClient httpClient) : base(httpClient)
+            builder.AppendToQuery("type", "artist");
+            builder.AppendToQuery("limit", limit);
+            if (after?.Length > 0)
+            {
+                builder.AppendToQuery("after", after);
+            }
+            return await GetModel<T>(builder.Uri, accessToken);
+        }
+        #endregion
+
+        #region UnfollowArtistsOrUsers
+        /// <summary>
+        /// Unfollow Artists
+        /// </summary>
+        /// <param name="artistIds">A comma-separated list of the artist Spotify IDs. A maximum of 50 IDs can be sent in one request. A minimum of 1 artist id is required.</param>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/unfollow-artists-users/
+        /// </remarks>
+        public async Task UnfollowArtists(
+            string[] artistIds = null,
+            string accessToken = null
+            )
+        {
+            if (artistIds?.Length < 1 || artistIds?.Length > 50) throw new
+                    ArgumentException("The artist ids can be a minimum of 1 and a maximum of 50.");
+
+            var builder = new UriBuilder($"{BaseUrl}/me/following");
+            builder.AppendToQuery("type", "artist");
+            builder.AppendToQueryAsCsv("ids", artistIds);
+            await Delete(builder.Uri, accessToken);
+        }
+
+        /// <summary>
+        /// Unfollow Users
+        /// </summary>
+        /// <param name="userIds">A comma-separated list of the user Spotify IDs. A maximum of 50 IDs can be sent in one request. A minimum of 1 user id is required. </param>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/unfollow-artists-users/
+        /// </remarks>
+        public async Task UnfollowUsers(
+            string[] userIds = null,
+            string accessToken = null
+            )
+        {
+            if (userIds?.Length < 1 || userIds?.Length > 50) throw new
+                    ArgumentException("The user ids can be a minimum of 1 and a maximum of 50.");
+
+            var builder = new UriBuilder($"{BaseUrl}/me/following");
+            builder.AppendToQuery("type", "user");
+            builder.AppendToQueryAsCsv("ids", userIds);
+            await Delete(builder.Uri, accessToken);
+        }
+        #endregion
+
+        #region UnfollowPlaylist
+        /// <summary>
+        /// Unfollow a Playlist
+        /// </summary>
+        /// <param name="playlistId">The Spotify ID of the playlist that is to be no longer followed.</param>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/unfollow-playlist/
+        /// </remarks>
+        public async Task UnfollowPlaylist(
+            string playlistId,
+            string accessToken = null
+            )
+        {
+            if (playlistId?.Length < 1) throw new
+                    ArgumentException("A valid playlist id has to be supplied.");
```

Comment:
See comment above. Should be:

    if (string.IsNullOrEmpty(playlistId)) throw new ArgumentNullException("playlistId");